client/asset/interface.go

```diff
@@ -270,6 +270,10 @@ type Wallet interface {
 	// payment. This method need not be supported by all assets. Those assets
 	// which do no support DEX registration fees will return an ErrUnsupported.
 	RegFeeConfirmations(ctx context.Context, coinID dex.Bytes) (confs uint32, err error)
+	// EstimateRegistrationTxFee provides an estimate for the tx fee needed to
+	// pay the registration fee. rateOracleFallback is used in the case where
```

Comment:
When this gets rebased after https://github.com/decred/dcrdex/pull/1430 is merged, please be sure to scour the comments for mentions of the old `rateOracleFallback`

Reply:
https://github.com/decred/dcrdex/pull/1430 is merged.  Sorry I held that up a few days longer than necessary.  I missed the update to that.

Reply:
All good, I'll get this ready to review soon.